### PR TITLE
Form section with UI implementation

### PR DESCRIPTION
**Pull request by Lino09**
This section was peer-programmed with my partners.
We added:

 - The form section
 - Validations for the form
 - Styles to the form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -128,54 +132,94 @@ <h3>Wootbit Software Develorper Agency</h3>
...
+                        <li>codepen</li>
+                        <li>Figma</li>
+                    </ul>
+                </li>
+            </ul>
+        </section>
+
+        <section class="form-section">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
```
Comment (ZeenatLawal) [inline index.html:187]: - [ ] Kindly adjust the spacing and font-size of this text.
- Your design
FireShot Capture 106 - Abel's Amazing Work - raw githack com


- Figma design

Screenshot (278)

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Lino09 , @Ntare22 and @micheaol, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Lino09) [reply index.html]: Thanks for your help

File: index.html [hunk 2]
```diff
@@ -128,54 +132,94 @@ <h3>Wootbit Software Develorper Agency</h3>
...
-            <li class="second-card">
-                <div>
-                    <img src="./assets/png/icon-frameworks.png" alt="">
-                </div>
-                <h3>Frameworks</h3>
+        <section id="about-me">
+            <div class="about-me-first">
+                <img id="top-right-img" src="./assets/svg/1-aboutme-mobile.svg" alt="">
+                <h2>About me</h2>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
```
Comment (Alexoid1) [inline index.html:139]: - [OPTIONAL] Kindly use the font family, font size, and color same as the Figma template.

File: index.html [hunk 3]
```diff
@@ -21,10 +24,11 @@
     <main class="landing">
         <div class="landiv">
             <h1>Hey there. I'm Abel<br><span class="greeny">I'm a software developer</span></h1>
-            <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work
```
Comment (Alexoid1) [inline index.html:27]: - [OPTIONAL] Kindly use the font-family, font-size and color same as the Figma template.

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @Lino09, @Ntare22 and @micheaol, Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! tada

# OPTIONAL
- [OPTIONAL] Kindly use the font-family, font-size and color same as the Figma template.

brent-rambo-thumbs-up


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**